plugins/destination/duckdb/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/cloudquery/codegen v0.3.19
 	github.com/cloudquery/plugin-sdk/v4 v4.60.0
 	github.com/google/uuid v1.6.0
-	github.com/marcboeker/go-duckdb v1.7.0
+	github.com/marcboeker/go-duckdb v1.7.2-0.20240830073305-42d190898463
```

Comment:
This includes a [fix](https://github.com/marcboeker/go-duckdb/commit/42d190898463ad2b407151ebc05156850fd29591) to have better error handling/messages during `appender.Close()`.